db/migrate/20170124220632_change_integer_limit_in_transaction.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeIntegerLimitInTransaction < ActiveRecord::Migration[5.0]
+  def change
+    change_column :transactions, :credit_card_number, :integer, limit: 8
```

Comment:
Not digits exactly, it refers to bits.  8 bits is what is required for 16 digits